### PR TITLE
Refactoring labels

### PR DESCRIPTION
**Pull request by nlamirault**
- Support Kubernetes recommanded labels like in Helm best practices
- Addtional labels to add into metadata of all resources.

### CODE REVIEW

**Comment by mhausenblas** [comment]
Happy to take a look, but might take a couple of days due to holiday season.

File: charts/adot-exporter-for-eks-on-ec2/values.yaml [hunk 1]
```diff
@@ -146,7 +151,7 @@ fluentbit:
     readFromTail: "On"
 
 fargateLog:
-  enabled: false
+  enabled: true
```
Comment (mhausenblas) [inline charts/adot-exporter-for-eks-on-ec2/values.yaml:154]: I'm not sure I follow why Fargate logging should be enabled in a context where the data plane is EC2?

File: charts/adot-exporter-for-eks-on-ec2/values.yaml [hunk 2]
```diff
@@ -4,6 +4,11 @@ global:
   namespaceOverride: ""
 awsRegion: ""
 clusterName: ""
+
+# -- Additional labels to add into metadata
+additionalLabels: #{}
```
Comment (mhausenblas) [inline charts/adot-exporter-for-eks-on-ec2/values.yaml:9]: Empty?

**Review by mhausenblas — COMMENTED** [review]
Overall LGTM, I left two comments inline. In addition: how are we making sure that this works as planned (testing)?

**Comment by mhausenblas** [comment]
@nlamirault apologies for the delay here, thanks for the contrib and looking forward to see this merged once the outstanding questions are addressed.

File: charts/adot-exporter-for-eks-on-ec2/values.yaml (hunk 1 above)
Comment (nlamirault) [reply charts/adot-exporter-for-eks-on-ec2/values.yaml]: Fixed. It was for test.

File: charts/adot-exporter-for-eks-on-ec2/values.yaml (hunk 2 above)
Comment (nlamirault) [reply charts/adot-exporter-for-eks-on-ec2/values.yaml]: Fixed. Sorry.

**Comment by mhausenblas** [comment]
Okay, thanks for the quick fixes and happy to be merged now @alolita

**Review by Aneurysm9 — APPROVED** [review]

**Pull request merged**